src/js/main.js

```diff
@@ -0,0 +1,88 @@
+import TabsManager, { TabItem } from './tabs.js';
+import UserApi from './api/user-api.js';
+
+//Carousel
+document.addEventListener('DOMContentLoaded', () => {
+    $('#carousel').slick({
+        infinite: true,
+        slidesToShow: 3,
+        slidesToScroll: 3,
+        prevArrow: $('.carousel-wrapper__left'),
+        nextArrow: $('.carousel-wrapper__right')
```

Comment:
`нужно поправить`

Стоит добавить адаптивность для слайдера. Для примера:
```
$('.responsive').slick({
  dots: true,
  infinite: false,
  speed: 300,
  slidesToShow: 4,
  slidesToScroll: 4,
  responsive: [
    {
      breakpoint: 1024,
      settings: {
        slidesToShow: 3,
        slidesToScroll: 3,
        infinite: true,
        dots: true
      }
    },
    {
      breakpoint: 600,
      settings: {
        slidesToShow: 2,
        slidesToScroll: 2
      }
    },
    {
      breakpoint: 480,
      settings: {
        slidesToShow: 1,
        slidesToScroll: 1
      }
    }
    // You can unslick at a given breakpoint now by adding:
    // settings: "unslick"
    // instead of a settings object
  ]
});
```